week-1-ts-ecosystem/w01e01-typescript-basics/src/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { IUser } from "./types";
+
+export function hasAddress(user: IUser): boolean {
+  return Boolean(user.address);
+}
+
+export function hasGivenAge(requiredAge: number): (user: IUser) => boolean {
+  return (user: IUser): boolean => user.age >= requiredAge;
+}
```

Comment:
tutaj standardowo doradzam żeby to nazwać `validators` bo to konkretny rodzaj utlisów :)